### PR TITLE
firebase-app-distribution 0.1.0

### PR DESCRIPTION
**Pull request by guness**
TagCheck

### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [x] __I will not move an already shared step version's tag to another commit__
- [x] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [x] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [x] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [x] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

### CODE REVIEW

File: steps/firebase-app-distribution/0.1.0/step.yml [hunk 1]
```diff
@@ -0,0 +1,108 @@
+title: '[Beta]Firebase App Distribution'
```
Comment (trapacska) [inline steps/firebase-app-distribution/0.1.0/step.yml]: please have an empty space between the texts and the `[Beta]` part as I am not sure if this way the users can find it by searhing for the firebase keyword.

Also, [BETA] might be more noticeable.

Comment (trapacska) [reply steps/firebase-app-distribution/0.1.0/step.yml]: apply the comment above for all of this texts

File: steps/firebase-app-distribution/0.1.0/step.yml [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+      Release notes for this build.
+    is_required: false
+    title: Release Notes
+  release_notes: null
+- opts:
+    description: |
+      Plain text file containing release notes for this build.
+    is_required: false
+    title: Release Notes File
+  release_notes_file: null
```
Comment (trapacska) [inline steps/firebase-app-distribution/0.1.0/step.yml]: both `_file` input makes not a very real use case on Bitrise. Please remove release_notes_file and testers_file and groups_file.

File: steps/firebase-app-distribution/0.1.0/step.yml [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+    is_required: false
+    summary: null
+    title: Test Groups
+- groups_file: null
+  opts:
+    description: |
+      Plain text file containing a comma-separated list of group names.
+    is_required: false
+    summary: null
+    title: Test Groups File
```
Comment (trapacska) [inline steps/firebase-app-distribution/0.1.0/step.yml]: please add an "Additional command flags" input, so the users are able to specify any extra flags they need (also the `_file` suffixed ones)

File: steps/firebase-app-distribution/0.1.0/step.yml (hunk 1 above)
Comment (guness) [reply steps/firebase-app-distribution/0.1.0/step.yml]: done

File: steps/firebase-app-distribution/0.1.0/step.yml (hunk 2 above)
Comment (guness) [reply steps/firebase-app-distribution/0.1.0/step.yml]: done

File: steps/firebase-app-distribution/0.1.0/step.yml (hunk 3 above)
Comment (guness) [reply steps/firebase-app-distribution/0.1.0/step.yml]: done

**Comment by trapacska** [comment]
Thank you for the step @guness ! 🚀 🎉

**Comment by rohan20** [comment]
(I may be asking at the wrong place, please redirect me to the right place otherwise) https://github.com/bitrise-io/bitrise-steplib/pull/2155/files#diff-3e5d14cb744d065472bb2c07a921dce4R14-R19 

Is this supported for Flutter builds too? I don't see it in the list of `project_type_tags`.

**Comment by guness** [comment]
@rohan20 flutter added already. you are looking at first version of the library:
here is the source:

https://github.com/guness/bitrise-step-firebase-app-distribution/blob/master/step.yml

**Pull request merged**